fastlane_core/lib/fastlane_core/ui/implementations/shell.rb

```diff
@@ -121,3 +121,9 @@ def verify_interactive!(message)
     end
   end
 end
+
+class String
```

Comment:
@KrauseFx `UI.deprecated` puts a timestamp in the front of the line which makes the output on `fastlane actions` and `fastlane action <action_name>` look super weird since nothing else as a timestamp in front of it 😉  

I also wanted `deprecated` as a common method (since its combination of a color and a bold style) so I could use it here - https://github.com/fastlane/fastlane/pull/7609/files#diff-43650ff42a4b937717605d848a2d8f7aR18